sw/device/tests/sim_dv/lc_walkthrough_test.c

```diff
@@ -24,6 +24,8 @@ static dif_otp_ctrl_t otp;
  * These tokens will be further randomized and overridden by the testbench.
  */
 
+static volatile const uint8_t kDestState = 0x0;
```

Comment:
Globals are defined to be zeroed bu default. You can drop the `0x0`.

Reply:
Thanks Miguel for pointing that out. Updated